test/unit/connection-parameters/creation-tests.js

```diff
@@ -67,6 +67,30 @@ test('ConnectionParameters initializing from config', function () {
   assert.ok(subject.isDomainSocket === false)
 })
 
+test('ConnectionParameters initializing from config and config.connectionString', function() {
+  var subject1 = new ConnectionParameters({
+    connectionString: 'postgres://test@host/db'
+  })
+  var subject2 = new ConnectionParameters({
+    connectionString: 'postgres://test@host/db?ssl=1'
+  })
+  var subject3 = new ConnectionParameters({
+    connectionString: 'postgres://test@host/db',
+    ssl: true
+  })
+  var subject4 = new ConnectionParameters({
+    connectionString: 'postgres://test@host/db?ssl=1',
+    ssl: false,
+    max: 5
```

Comment:
Is this supposed to work? Connection strings seem to work at the client level, whereas `max` is at the pool level.

If it’s not, it should probably be removed to avoid misleading people.